src/styles/calendar.scss

```diff
@@ -0,0 +1,64 @@
+@import "./variables.scss";
```

Comment:
This is working, but for my opinion it is better to imprort all files to `main.scss`. Because when you will have more files, there will be a chance to forget import some special file to separete file.

Reply:
Agree, it'd be better to import variables to main.scss